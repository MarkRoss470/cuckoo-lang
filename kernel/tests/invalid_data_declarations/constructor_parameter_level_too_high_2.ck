data Invalid.{u} : Type where
  | c : Sort u -> Invalid
