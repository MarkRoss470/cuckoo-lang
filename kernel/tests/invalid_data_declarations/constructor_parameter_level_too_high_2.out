ERROR at 2:9-2:16: Invalid level for constructor parameter - this parameter is of type
  Sort u
at level
  (u + 1)
which is not less than or equal to the inductive type's level
  1
